src/styles/blocks/stars4.css

```diff
@@ -0,0 +1,32 @@
+.stars--4 {
+  display: flex;
+  justify-content: space-between;
+  width: 96px;
+  margin-bottom: 4px;
+}
+
+.stars--4 div {
+  background-size: cover;
+  height: 16px;
+  width: 16px;
+}
+
+.stars--4 div:nth-child(1) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars--4 div:nth-child(2) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars--4 div:nth-child(3) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars--4 div:nth-child(4) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars--4 div:nth-child(5) {
+  background-image: url(../../images/star.svg);
+}
```

Comment:
The class `.stars--4` and its child divs are not following the BEM methodology correctly. Consider using a more appropriate BEM naming convention, such as `.stars__item--4`, and apply styles using pseudo-selectors instead of targeting each child individually.